epictrack-api/src/api/resources/event.py

```diff
@@ -100,22 +99,6 @@ def delete(event_id):
         return message, HTTPStatus.OK
 
 
-@cors_preflight("DELETE")
-@API.route("/events", methods=["DELETE", "OPTIONS"])
-class MilestoneEvents(Resource):
-    """Endpoint resource to manage milestone events"""
-
-    @staticmethod
-    @cors.crossdomain(origin="*")
```

Comment:
Are you deleting this because we are not using it at all?

Reply:
yup, no calls for it in the UI

Reply:
@jadmsaadaot Lets keep this method cz we might call it from UI in future for some other roles or we might use this api from postman to get stuff done at some point. So better we keep it.